test/functional/rpc_blockchain.py

```
@@ -202,7 +190,14 @@ def _test_getblockchaininfo(self):
         assert res['automatic_pruning']
         assert_equal(res['prune_target_size'], 576716800)
         assert_greater_than(res['size_on_disk'], 0)
-        assert_equal(res['softforks'], {
+
+    def check_signalling_deploymentinfo_result(self, gdi_result, height, blockhash, status_next):
+        assert height >= 144 and height <= 287
+
+        assert_equal(gdi_result, {
+          "hash": blockhash,
```

Flake8 (7.3.0)

[error] continuation line under-indented for hanging indent

(E121)